### ISSUE TITLE
Make it more clear that the "forced" license is not like all other licenses

### ISSUE THREAD

**Issue by senff** [Automattic/woocommerce-payments#5214]
### Description
Whenever a site has WooCommerce Payments installed (active or not), it will show in the list of licenses (under **WooCommerce → Extensions → WooCommerce.com Subscriptions**) as an active, lifetime license (due to https://github.com/Automattic/woocommerce.com/pull/10613 ).  However, even though this is a "special" license, it looks like every other (paid) license, so it's confusing to users.

### To Reproduce
1. Make sure you're logged in to WooCommerce.com
2. Make sure the WooCommerce Payments plugin is installed on your site.
3. Go to **WooCommerce → Extensions → WooCommerce.com Subscriptions**
4. Try to deactivate the license for WooCommerce Payments.
5. Error message shows: "_An error has occurred when deactivating the subscription for WooCommerce Payments. Please try again later._"

### Suggested options
The error message should clarify that this is a "special" plugin/license, that can not be activated (for example "_A license for WooCommerce Payments is included with the plugin and can not be deactivated._") Since it's by **design** that the license can not be deactivated, it should not be presented as an **error**.

It might also be helpful to indicate that this license is different from a "regular" license.  Rough idea:

<img width="1086" alt="license" src="https://user-images.githubusercontent.com/1710669/204882489-6da393b3-0fc9-435e-ae7e-e456ccfccea0.png">

Or, remove this license altogether (as it is free and forever and therefore just a standard feature of WooCommerce.com)

**Basically, if we are going to "force" this license onto customers, we should at least make it clear that it's not like all the other licenses.**

**Comment by jbordonado** [comment]
@senff Wouldn't that be on [WooCommerce.com](https://github.com/Automattic/woocommerce.com) side then? I don't think we have control over this in our codebase. Can you suggest this in their repo?

**Comment by senff** [comment]
@jbordonado Right, this may be more of an issue for them, I'll post it there. Thanks!
